copydb/copydb.go

```diff
@@ -109,11 +112,22 @@ func (this *CopydbFerry) Run() {
 	// should be identical.
 	copyWG.Wait()
 
-	logrus.Info("ghostferry main operations has terminated but the control server remains online")
-	logrus.Info("press CTRL+C or send an interrupt to stop the control server and end this process")
 	// This is where you cutover from using the source database to
 	// using the target database.
 
+	// We have to listen to signals because the Ghostferry builtin signal handler
+	// has quit but this process is still alive
+	go func() {
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Does this not override the ghostferry signal handlers? Is there some gopher magic to register multiple signal handlers?

Reply:
I changed the base signal handler so it doesn't quit after Ferry.Run is finished. Based on if the context in Ferry.Run is canceled or not, it will either exit the process normally or panic.